pkg/users/accounts-list.js

```diff
@@ -145,6 +145,7 @@ const getGroupRow = (group, accounts) => {
 };
 
 const getAccountRow = (account, current, groups) => {
+    console.log(current);
```

Comment:
Accidentally left in?

Reply:
Right, let's remove that. Otherwise LGTM, thanks!